subsys/usb/usb_c/usbc_pe_common.c

```diff
@@ -19,6 +19,20 @@ LOG_MODULE_DECLARE(usbc_stack, CONFIG_USBC_STACK_LOG_LEVEL);
 
 static const struct smf_state pe_states[PE_STATE_COUNT];
 
+/**
+ * @brief Set the ready state for sink or source.
+ */
+static void pe_set_ready_state(const struct device *dev)
+{
```

Comment:
Please update the commit description to better describe the fix.

"Policy engine errors were unconditionally setting the state back to sink ready.  This fix sets the correct state based on the current power role."

Reply:
done